packages/ui/src/Text/index.tsx

```diff
@@ -102,7 +102,9 @@ const getTextOptionClasses = ({
   break: breakProp,
   whitespace,
 }: TextProps): string => {
-  const truncateClass = truncate ? cn('truncate') : '';
+  const truncateClass = truncate
+    ? cn('truncate max-w-[150px] overflow-hidden whitespace-nowrap')
+    : ''; // Adds max-width for aggressive truncation
```

Comment:
what defines the truncations max width?

<img width="245" alt="Screenshot 2024-12-10 at 10 47 16 AM" src="https://github.com/user-attachments/assets/9837f9f6-b472-4e41-a8df-72787a93059a">